test/exec/continuous-integration.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+npm install
+
+# Setup the virtual frame buffer and export the DISPLAY env variable
+Xvfb :1 -screen 5 1024x768x8 &
```

Comment:
Schweet! Our ubuntu machines have this (selenium grid) but not sure if ubreakit does without having to install it, do we need to add something in server-templates?